lib/Factory/LayoutFactory.php

```diff
@@ -1425,7 +1425,7 @@ public function createFromZip(
                         $upgraded = $widgetCompatibilityInterface->upgradeWidget($widget, $widget->schemaVersion, 2);
                         if ($upgraded) {
                             $widget->schemaVersion = 2;
-                            $widget->save(['alwaysUpdate'=>true]);
+                            $widget->save(['alwaysUpdate'=>true, notifyDisplays => false]);
```

Comment:
You're missing some quotes around that string 😄

Reply:
Sorry :(